src/components/card/card.jsx

```diff
@@ -0,0 +1,73 @@
+//import { useState } from 'react';
+import './card.styles.css';
+const Card = (cardsData) => {
```

Comment:
Is not the convention for writing a react component is 
`const Component = (props) => {`

Using a different variable name causes confusion for other devs. Adhere to the convention.

Reply:
Or you can destructure it directly there so that you do not need to write `cardsData.cardsData` everywhere.